components/form-builder/app/Start.tsx

```diff
@@ -112,22 +112,24 @@ export const Start = () => {
           <h2 className="p-0 mb-1 group-hover:underline group-focus:underline">{t("startH2")}</h2>
           <p className="text-sm">{t("startP1")}</p>
         </button>
-        <label
```

Comment:
The problem was the invisible input referencing a visible label. The fix is just nesting the input which arguably is still referencing a visible input but... it will now past WCAG automated test tools.
Also making a label interactive will make some but not all WCAG automated test tools grumpy. The fix is just to make a div interactive instead.
Oh and the ARIA spec requires that button be can be keyboard activated by both ENTER and SPACE keys